src/components/Public/Record/Importer/FieldMatch.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <b-card header-bg-variant="white"
+          footer-bg-variant="white">
+
+    <b-container fluid
+                 class="p-0">
+
+      <b-form-group>
+        <label>{{ $t('block.recordList.import.matchFields') }}</label>
+        <b-table small
+                 :fields="tableFields"
+                 :items="rows">
+
+          <template slot="HEAD[selected]">
+            <b-form-checkbox :checked="selectAll"
+                             @change="onSelectAll" />
+          </template>
+
+          <template slot="[selected]"
+                    slot-scope="data">
+
+            <b-form-checkbox v-model="data.item.selected" />
+          </template>
+
+          <template slot="[moduleField]"
+                    slot-scope="data">
+
+            <b-form-select v-model="data.item.moduleField"
+                           :options="moduleFields">
+
+              <template slot="first">
+                <option :value="undefined"
+                        disabled>
+
+                  {{ $t('block.recordList.import.pickModuleField') }}
+                </option>
+              </template>
+            </b-form-select>
+          </template>
+        </b-table>
+
+      </b-form-group>
+    </b-container>
+
+    <div slot="footer"
+         class="d-flex">
+
+      <span class="ml-auto">
```

Comment:
why wrapping button in span? just add float-right class to button and remove span